pkg/kustomize/kustomize_cli.go

```diff
@@ -49,3 +56,36 @@ func extractSemanticVersion(version string) (string, error) {
 	}
 	return regex.FindString(version), nil
 }
+
+// HasKustomize finds out if there is any kustomize resource in the cwd or subdirectories
+func (k *KustomizeCLI) HasKustomize() bool {
+	if len(k.FindKustomize()) != 0 {
+		return true
+	}
+
+	return false
+}
+
+// FindKustomize looks for the kustomization.yaml i.e. kustomize resources in present and sub-directories
+func (k *KustomizeCLI) FindKustomize() (resource []string) {
```

Comment:
Do you really need to export this function? Is this not just a helper for HasKustomize?

Reply:
I planned this in order to fetch paths of all customisation files so that we know which all directories to apply kustomize to.